### PR TITLE
Add a additional steps (which is flash and add a certification file) on the ST_Things SDK document

### PR DESCRIPTION
**Pull request by sojinp**
- Add additional steps to flash a binary file to board on the 'HowToDevelopThingsWithTizenStudioforRT`.
Upload the media files about flash.

- Modified '4.Build and Flash the Project'(that is title of build steps) to '4.Build the Project' on the HowToDevelopThingsWithTizenStudioForRT.md

- Add additional steps to add a certificate file on the 'HowToDevelopThingsUsingSmartThingsThingsSDK.md'.
This steps added on the Prerequisites' steps.

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

File: docs/HowToDevelopThingsUsingSmartThingsThingsSDK.md [hunk 1]
```diff
@@ -20,6 +20,12 @@ Then, follow the steps.
 1. Get your MNID - Manufacturer ID - on [My Page](https://smartthings.developer.samsung.com/partner/dashboard)  
 2. Create a cloud-connected device [[Details]](https://smartthings.developer.samsung.com/develop/workspace/ide/create-a-cloud-connected-device.html)  
 3. Generate the Certificate Signing Request and issue a new Signing key for the device certificate [[Details]](https://smartthings.developer.samsung.com/develop/workspace/ide/about-common-tools.html)  
+4. Add `certificate.pem` and `privateKey.der` file to `$TIZENRT_BASEDIR/tools/fs/contents/`   
+5. Edit Json file to match the following: json
```
Comment (JeongJunSik) [inline docs/HowToDevelopThingsUsingSmartThingsThingsSDK.md]: Please change "json" to "JSON"

File: docs/HowToDevelopThingsUsingSmartThingsThingsSDK.md [hunk 2]
```diff
@@ -20,6 +20,12 @@ Then, follow the steps.
 1. Get your MNID - Manufacturer ID - on [My Page](https://smartthings.developer.samsung.com/partner/dashboard)  
 2. Create a cloud-connected device [[Details]](https://smartthings.developer.samsung.com/develop/workspace/ide/create-a-cloud-connected-device.html)  
 3. Generate the Certificate Signing Request and issue a new Signing key for the device certificate [[Details]](https://smartthings.developer.samsung.com/develop/workspace/ide/about-common-tools.html)  
+4. Add `certificate.pem` and `privateKey.der` file to `$TIZENRT_BASEDIR/tools/fs/contents/`
```
Comment (JeongJunSik) [inline docs/HowToDevelopThingsUsingSmartThingsThingsSDK.md]: please change "file" to "files"

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request closed without merging**